e2e/README.md

```diff
@@ -22,11 +22,53 @@ nodes can be identified by the following container names (kube-master, kube-node
 All tests with the name `SmokeSuite` in the method will execute and the results will be of a junit
 type output to the `e2e/results` directory.
 
+## Run Tests
+Rook Test Framework can be used to set up rook infrastructure and then run tests, or it can be used to run tests against an existing/pre-installed rook ,when users my choose to install kubernetes and rook using their own flavor.  The difference between running test directly and using ```e2e/scripts/smoke_test.sh``` is that rook-test container is not set up. When running tests directly a Kubernetes three node cluster is created using Kubeadm on the Host directly. 
+  
+#### Test paramaters
+The following parameters are available while running tests
+
+ Parameter | Description | Default value
```

Comment:
I think it would still be useful to show possible values even if there is only one possibility. Otherwise people will wonder why they can't run against a different version of kubernetes, etc.